src/Components/Admin/ObjectEditor/object.js

```diff
@@ -161,6 +179,24 @@ export default function ObjectEditor({
         setIsInteractable(!isInteractable);
     };
 
+    const addHeader = () => {
+        const newText = {
+            text: prompt("Enter the text for the header: "),
+        };
+
+        if (newText.text) {
+            if (newText.text.length <= 100) {
+                setHeader(newText.text);
+            } else {
+                alert("Error: Maximum header text length is 100 characters");
```

Comment:
When we move away from `prompt` for getting input from admin, we can also replace this `alert` with a proper error text below the input field in the popup. For now, this is fine.